config/samples/cassandra.datastax.com_v1beta1_cassandradatacenter.yaml

```diff
@@ -1,7 +1,29 @@
 apiVersion: cassandra.datastax.com/v1beta1
 kind: CassandraDatacenter
 metadata:
-  name: cassandradatacenter-sample
+  name: dc2
 spec:
-  # Add fields here
-  foo: bar
+  clusterName: cluster2
+  serverType: cassandra
+  serverVersion: "3.11.11"
```

Comment:
Let's consider getting this updated to the 4.x line.

Reply:
Agreed.